orbital-mechanics/TheRealLecture1/SpacecraftDynamicsWeek1r;$.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>Not sure why, but there seems to be an extra “/“ in front of every equation. Maybe this is how it was typed out? Or maybe just me?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>More information on the equations throughout the notebook:</p><ul><li>Write the equations on each line (not a new cell, just leave some space between the equations and other texts.) For example:</li></ul>

<pre class="ql-syntax" spellcheck="false">### Two-Body Relative Dynamics


#### Newton's Second Law
1. $ m_2 \ddot{\vec{r}}_2 = \vec{F}_{21} = - &nbsp;\frac{G m_1 m_2 \vec{r}}{r^3} &nbsp;\rightarrow &nbsp;\ddot{\vec{r}}_2 = - &nbsp;\frac{G m_1 &nbsp;\vec{r}}{r^3} $


2. $ m_1 \ddot{\vec{r}}_1 = \vec{F}_{12} = &nbsp;\frac{G m_1 m_2 \vec{r}}{r^3} \rightarrow &nbsp;\ddot{\vec{r}}_2 = &nbsp;\frac{G m_2 &nbsp;\vec{r}}{r^3} $


Combining these equations:
</pre><ul><li>The equations need to have labels and they also need to be formatted as per this link: <a href="https://jupyterbook.org/en/stable/content/math.html#numbering-equations" target="_blank">https://jupyterbook.org/en/stable/content/math.html#numbering-equations</a></li><li>Follow the formatting style on this link for all the equations you have. </li></ul>